app/v1/controllers/requestParsers/validators/CreateAndAmendForeignReliefsValidator.scala

```diff
@@ -16,20 +16,22 @@
 
 package v1.controllers.requestParsers.validators
 
-import v1.controllers.requestParsers.validators.validations._
-import v1.models.errors.{MtdError, RuleIncorrectOrEmptyBodyError}
+import api.controllers.requestParsers.validators.Validator
+import api.controllers.requestParsers.validators.validations._
+import api.models.errors.{MtdError, RuleIncorrectOrEmptyBodyError}
 import v1.models.request.createAndAmendForeignReliefs._
 
 class CreateAndAmendForeignReliefsValidator extends Validator[CreateAndAmendForeignReliefsRawData] {
 
   private val validationSet = List(parameterFormatValidation, parameterRuleValidation, bodyFormatValidation, bodyFieldValidation)
 
-  private def parameterFormatValidation: CreateAndAmendForeignReliefsRawData => List[List[MtdError]] = (data: CreateAndAmendForeignReliefsRawData) => {
-    List(
-      NinoValidation.validate(data.nino),
-      TaxYearValidation.validate(data.taxYear)
-    )
-  }
+  private def parameterFormatValidation: CreateAndAmendForeignReliefsRawData => List[List[MtdError]] = (data: CreateAndAmendForeignReliefsRawData) =>
+    {
```

Comment:
it's a bit non-standard to have the curly braces on the next line like this. tbh they're not needed at all, could just delete them.
Because the 'def' line is a bit long, could move the stuff after "=" to the next line. i.e.
```
  private def parameterFormatValidation: CreateAndAmendForeignReliefsRawData => List[List[MtdError]] =
    (data: CreateAndAmendForeignReliefsRawData) =>
      List(
        NinoValidation.validate(data.nino),
        TaxYearValidation.validate(data.taxYear)
      )
```